### PR TITLE
UI Tweaks

### PR DESCRIPTION
**Pull request by Nodrex**

### CODE REVIEW

File: testomania/src/main/java/com/earth/testomania/presentation/AboutBottomSheet.kt [hunk 1]
```diff
@@ -72,6 +82,25 @@ fun AboutBottomSheet() {
...
         AboutDeveloper(it)
     }
+
+    Box(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(bottom = 10.dp),
+        contentAlignment = Alignment.Center
+    ) {
+        Icon(
```
Comment (shalva97) [inline testomania/src/main/java/com/earth/testomania/presentation/AboutBottomSheet.kt:92]: Why not just the Icon with .fillMaxWisth?

**Review by shalva97 — CHANGES_REQUESTED** [review]

**Comment by shalva97** [comment]
nice

File: testomania/src/main/java/com/earth/testomania/presentation/AboutBottomSheet.kt (hunk 1 above)
Comment (Nodrex) [reply testomania/src/main/java/com/earth/testomania/presentation/AboutBottomSheet.kt:92]: Ok let me try

Comment (Nodrex) [reply testomania/src/main/java/com/earth/testomania/presentation/AboutBottomSheet.kt:92]: yep works with fillMaxWidth, but it click now is clickable whole width and click animation is also stretched
image

Comment (Nodrex) [reply testomania/src/main/java/com/earth/testomania/presentation/AboutBottomSheet.kt]: Disable ripple animation, now it works

**Review by shalva97 — APPROVED** [review]

**Pull request merged**